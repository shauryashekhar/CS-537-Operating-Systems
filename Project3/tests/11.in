ls tests/p2a-test > ./tests-out/output11
cat ./tests-out/output11
rm -f ./tests-out/output11
exit
