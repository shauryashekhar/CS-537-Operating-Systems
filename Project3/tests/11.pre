echo "ls tests/p2a-test > ./tests-out/output11" > tests/11.out 2> tests/11.err
ls tests/p2a-test > ./tests-out/output11 2> ./tests-out/output11
echo "cat ./tests-out/output11" >> tests/11.out 2>> tests/11.err
cat ./tests-out/output11 >> tests/11.out 2>> tests/11.err
echo "rm -f ./tests-out/output11" >> tests/11.out 2>> tests/11.err
rm -f ./tests-out/output11 >> tests/11.out 2>> tests/11.err
echo "exit" >> tests/11.out 2>> tests/11.err

