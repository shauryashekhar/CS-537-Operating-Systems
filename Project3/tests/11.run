./mysh tests/11.in
