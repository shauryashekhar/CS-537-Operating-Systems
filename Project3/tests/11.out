ls tests/p2a-test > ./tests-out/output11
cat ./tests-out/output11
test1
test2
test3
test4
rm -f ./tests-out/output11
exit
